scripts/entry.js

```diff
@@ -70,7 +70,19 @@ function Entry(data)
     if(this.media){
       var parts = this.media.split(".")
       if (parts.length === 1) { this.media += ".jpg" } // support og media uploads
-      html += "<img class='media' src='"+this.dat+"/media/content/"+this.media+"'/>";
+      switch(parts[1]){
```

Comment:
For `my.vid.mp4` this should be `parts[parts.length-1]` ?

Reply:
oh, well spotted, thanks!